rootfs/usr/bin/chimera-session

```diff
@@ -66,25 +69,28 @@ function desktop_xorg() {
 function gamepadui() {
 	# switch to Gamepad/Deck UI (Gamescope/Wayland)
 	echo '' > ${SESSION_CONFIG}
+	clean_steam_shortcuts
+	ln -s /usr/share/return-sessions/org.chimeraos.Gamescope.desktop ${DATA_HOME}/applications/session-return.desktop
```

Comment:
`rg.chimeraos.Gamescope.desktop` < should be lowercase G?

Reply:
I left the case as it was, but we can change it